index.html

```diff
@@ -0,0 +1,525 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Space Ripples Clone</title>
+  <link rel="stylesheet" href="./css/style.css">
+  <link rel="shortcut icon" href="./Images/nyt.png" type="image/x-icon">
+  <!--
+  <link rel="stylesheet" href="./css/font-awesome-4.7.0/css/font-awesome.min.css">
+  -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  
+</head>
+<body>
+  <nav id="navbar">
+    <div class="navbar-section" id="left">
+      <i class="fa fa-bars icons" aria-hidden="true"></i> 
+      <i class="fa fa-search icons" aria-hidden="true"></i>
+    </div>
+
+    <div class="navbar-section" id="middle">
+      <img src="https://www.goyonder.it/wp-content/uploads/2019/09/2a9.f63.NYT-large-black-clear-background.png" alt="new-york-times" class="navbar-logo">
+    </div>
+
+    <div class="navbar-section" id="right">
+      <button class="subscribe" type="button">SUBSCRIBE NOW</button>
+      <span class="account">
+        Account
+        <i class="fa fa-chevron-circle-down" aria-hidden="true"></i>
+      </span>
+    </div>
+  </nav>
+
+  <main>
+    <header>
+      <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+      <figure>
+        <img class="header-image" src="./Images/guy-1.jpg" alt="Picture of Alan Guth" />
+        <figcaption>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.<span class="author-name">Rick Friedman for The New York Times</span></figcaption>
+      </figure>  
+    </header>
+    <article class="content">
+      <section class="share">
+        <h3>By Dennis Overbye</h3>
+        <div class="date-icons">
+          <div>
+            March 17, 2014
+          </div>
+          <div class="social-icons">
+            <div class="icon-button facebook"></div>
+            <div class="icon-button twitter"></div>
+            <div class="icon-button envelope"></div>
+            <div class="icon-button share-content"></div>
+            <div class="bookmark-comments">  
+              <i class="fa fa-bookmark-o fa-2x" style="margin: 0 2%" aria-hidden="true"></i> 
```

Comment:
- [ ] **MANDATORY** For good coding practices, don't use inline styling, please put this styles in your css file
![image](https://user-images.githubusercontent.com/53256139/87738096-3b324100-c7a2-11ea-977b-ede338d3191f.png)